lib/vinyldns/api.rb

```diff
@@ -53,7 +53,13 @@ def self.make_request(signed_object, uri, body = '')
       url = URI(signed_object.api_url)
       https = Net::HTTP.new(url.host, url.port)
       https.use_ssl = true ? url.scheme == "https" : https.use_ssl = false
-      https.verify_mode = OpenSSL::SSL::VERIFY_NONE # SSL not signed? Don't care!
+      if ENV['VINYLDNS_VERIFY_SSL'] || ENV['VINYLDNS_VERIFY_SSL'] == 'true' || ENV['VINYLDNS_VERIFY_SSL'].nil?
+        https.verify_mode = OpenSSL::SSL::VERIFY_PEER
+      elsif !ENV['VINYLDNS_VERIFY_SSL'] || ENV['VINYLDNS_VERIFY_SSL'] == 'false'
+        https.verify_mode = OpenSSL::SSL::VERIFY_NONE
+      else
+        raise('Unsupported value for ENV[\'VINYLDNS_VERIFY_SSL\']!')
+      end
```

Comment:
I think you need a test for this. The if and elsif seem to be covering all cases with `if ENV['VINYLDNS_VERIFY_SSL']` and `elsif !ENV['VINYLDNS_VERIFY_SSL']` so you would never hit the final else. Also it may be cleaner to put this logic in a separate method.

Reply:
still think the simplest approach would be to only look for the environment variable set to 'false', anything else leaves it enabled.

Reply:
☝️ I agree with @britneywright here. And to cover your bases, you could check for `ENV['VINYLDNS_SSL_VERIFY'] == false || ENV['VINYLDNS_SSL_VERIFY'].to_s.downcase == "false"` on the off chance that someone has...

* `ENV['VINYLDNS_SSL_VERIFY'] = false` from within Ruby or somehow coerced "false" to `false`
*  `export VINYLDNS_SSL_VERIFY=FALSE` or some set it to some casing other than "false"